Classes/Emogrifier.php

```diff
@@ -1501,4 +1518,37 @@ private function getNodesToExclude(\DOMXPath $xPath)
 
         return $excludedNodes;
     }
+
+    /**
+     * Handles invalid xPath expression warnings, generated by process() method,
+     * during querying \DOMDocument and trigger \InvalidArgumentException
+     * with invalid selector
+     *
+     * @param int $type
+     * @param string $message
+     * @param string $file
+     * @param int $line
+     * @param array $context
+     *
+     * @return bool always false
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function handleXpathError($type, $message, $file, $line, array $context)
```

Comment:
It's have to be public, it's callback. But there is one option. Callback can be hidden to directly to the process like `set_error_handler(function() { }, ...);` but that's avoid options to overwrite handler behavior.